packages/combo-box/test/keyboard.test.js

```diff
@@ -69,6 +69,25 @@ describe('keyboard', () => {
 
       expect(getFocusedIndex()).to.equal(0);
     });
+
+    it('should propagate escape key event if dropdown is closed', () => {
+      const keyDownSpy = sinon.spy();
+      document.body.addEventListener('keydown', keyDownSpy);
+      escKeyDown(input);
+      document.body.removeEventListener('keydown', keyDownSpy);
```

Comment:
I would suggest an explicit assertion for consistency with the other codebase.

```suggestion
      const event = keyboardEventFor('keydown', 27, [], 'Escape');
      const keyDownSpy = sinon.spy(event, 'stopPropagation');
      input.dispatchEvent(event);
```